src/app/forms-builder/components/dynamic-fields/button/button.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+import { FieldConfig } from '../../../../core/interfaces/field.interface';
+import { SubclassComponent } from '../../../../core/interfaces/subclass.component';
+
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html'
```

Comment:
stylesUrls must be present by default, even it's empty
```suggestion
  selector: 'app-button',
  templateUrl: './button.component.html',
  styleUrls: ['./button.component.scss']
```